### DISCUSSION TITLE
Where are sourcemaps located?

### DISCUSSION THREAD

**Discussion by N00nDay** [sveltejs/kit#5200] [Q&A]
I cannot find them anywhere, I don't even see that they are being created when I run `npm run build`. I am using the latest version of sveltekit with typescript. I would assume the sourcemaps would be located in the output folder of the .svelte-kit directory but no dice.

```
// svelte.config.js
...
const config = {
   compilerOptions: {
      enableSourcemap: true
   },
   preprocess: [
      preprocess({
         postcss: true,
         sourceMap: true
      })
   ],
   kit: {
      adapter: vercel(),
      methodOverride: {
         allowed: ['PATCH', 'DELETE']
      },
   }
};

export default config;
```

I also have `sourceMap: true` under the `compilerOptions` of my `tsconfig.json`

**Comment by vhscom** [comment]
In more recent versions of SvelteKit, the vite config specifies `sourcemap` option under `build`. I believe that replaces the `compilerOptions.enableSourcemap`. Related question on SO: https://stackoverflow.com/questions/72620350/where-are-sourcempas-located-and-how-do-i-generate-them-in-sveltekit

**Comment by rohanrajpal** [comment]
Also curious about this, I tried to upload sourcemaps to sentry for Sveltekit a few days ago but failed to do so. Might write a blog when i figure it out.

**Reply by half2me** [reply]
Please do @rohanrajpal, as I have been struggling with the same problem :)

**Reply by rohanrajpal** [reply]
yes definitely @half2me 

So far this solution seems to be the best, but i'm yet to try it out
https://github.com/getsentry/sentry-javascript/discussions/5838#discussioncomment-3895915

**Reply by flavio-greenfact** [reply]
I am using the @sentry/vite-plugin and I was able to upload the sourcemaps
